### ISSUE TITLE
List of included items by the version in which they were included in Python.

### ISSUE THREAD

**Issue by nuno-andre** [python/typing_extensions#299]
This list, [formerly included in the readme](https://github.com/python/typing_extensions/blob/962936a9e4583d4d424ff85745e9aeb6cf4460ee/README.md), is extremely useful. Especially when developing on a later version than the minimum supported by the code.

Would you consider adding it back into the docs?

**Comment by JelleZijlstra** [comment]
Yes, we can add a section for this in https://typing-extensions.readthedocs.io/en/latest/

**Comment by nuno-andre** [comment]
Happy to help with this. Could you provide a suitable format?

**Comment by JelleZijlstra** [comment]
Open to suggestions from others, but one approach would be to have a table with four columns:
- Name of the object (cross-linked to its docs entry)
- Python version when it was added to the standard library
- Version when it was added to typing-extensions (if it's in the changelog, no need to figure out the exact versions before 4.0 when we didn't keep a changelog)
- PEP that added the feature (if any)

**Comment by AlexWaygood** [comment]
> one approach would be to have a table with four columns

That sounds good to me. I'd vote for us using a [list-table](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive); they're much easier to maintain than the other style.
